### PR TITLE
Fix Client.config example in README

### PR DESCRIPTION
**Pull request by Bpless**

### CODE REVIEW

**Comment by Bpless** [comment]
Closes #49

File: README.md [hunk 1]
```diff
@@ -78,11 +78,13 @@ import json
 
 from plaid import Client
 
-client = Client.config({
+Client.config({
     'url': 'https://tartan.plaid.com',
     'suppress_http_errors': True,
 })
```
Comment (davidchambers) [inline README.md]: `Client.config` mutates `Client`? Yikes!

**Comment by davidchambers** [comment]
:deciduous_tree:

File: README.md (hunk 1 above)
Comment (Bpless) [inline README.md]: It's not ideal and definitely something that can be improved in the future.

**Pull request merged**